sputnikdao2/src/upgrade.rs

```diff
@@ -11,8 +11,11 @@ const UPDATE_GAS_LEFTOVER: Gas = Gas(10_000_000_000_000);
 const FACTORY_UPDATE_GAS_LEFTOVER: Gas = Gas(15_000_000_000_000);
 const NO_DEPOSIT: Balance = 0;
 
-pub const GAS_FOR_UPGRADE_SELF_DEPLOY: Gas = Gas(15_000_000_000_000);
-pub const GAS_FOR_UPGRADE_REMOTE_DEPLOY: Gas = Gas(15_000_000_000_000);
+pub const GAS_FOR_UPGRADE_SELF_PROMISE_CREATION: Gas = Gas(15_000_000_000_000);
+pub const GAS_FOR_UPGRADE_REMOTE_PROMISE_CREATION: Gas = Gas(15_000_000_000_000);
+/// Since Nightshade V2, the send_not_sir of action_function_call_per_byte increase to this value, please refer to: 
+/// https://github.com/near/nearcore/blob/0c2374993fc74b57faf2bcdf5c7c73a37e82b75a/core/parameters/res/runtime_configs/parameters.snap#L52
+pub const GAS_FUNCTION_CALL_PER_BYTE: u64 = 47_683_715;
```

Comment:
I think we can update the constant to be more precise:

The gas cost per byte for cross-shard function call action is `47_683_715` (`send_not_sir`) + `2_235_934` (`execution`)  = `49_919_649`, close to 50TB per MB.

Reply:
Oh, my bad, after second thought, here we should only charge the fees for sending the receipt, and don't need to include the execution gas cost, so I think the gas cost estimation is OK.